.env.example

```diff
@@ -4,4 +4,8 @@ OVISBOT_HTB_CREDS_EMAIL=htb@randomdomain.com
 OVISBOT_HTB_CREDS_PASS=myhtbpassword
 OVISBOT_HTB_TEAM_ID=1000
 OVISBOT_CTFTIME_TEAM_ID=999
-OVISBOT_ADMIN_ROLE=moderator
\ No newline at end of file
+OVISBOT_ADMIN_ROLE=moderator
+OVISBOT_DB_URL=mongodb://localhost/ovisdb
+OVISBOT_THIRD_PARTY_COGS_INSTALL_DIR=~/cogs
+OVISBOT_GITHUB_TOKEN=<github_token>
+OVISBOT_GITHUB_REPO_API_PATH=<github_solves_repo>
```

Comment:
Let's decide and set this before merging

Reply:
My input for this is to break it down to `OVISBOT_GITHUB_USER` and  `OVISBOT_GITHUB_REPO`. 
The user should not be responsible for providing the full API path; we just need the absolute necessary info. It was a bad, initial, design decision from my side.

Reply:
I have changed this and broke down the parameters to the necessary ones.